svd-rs/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
 - skip serializing `values` in `EnumeratedValues` if empty
+- rename `SingleArray` to `MaybeArray`, add `names` function for arrays
```

Comment:
you can just rename the entry for SingleArray directly, instead of having two changelog entries for it

Reply:
Done